lib/types.py

```diff
@@ -181,6 +181,10 @@ def parse_uint64(self, x, type):
         assert(0 <= x < (1 << 64))
         return fg3(str(x))
 
+    def parse_bool(self, x, type):
+        assert(0 <= x < (1 << 2))
```

Comment:
Thanks for the fix! I think this should be `0 <= x < (1 << 1))`?

Reply:
Ooops! Yeah, that is correct!

Reply:
Would you like me to close the pull request and submit a new one?